service/task.go

```diff
@@ -911,7 +914,11 @@ func (uis *UIServer) getTestResults(w http.ResponseWriter, r *http.Request, proj
 				et, err = task.FindOneId(t)
 			}
 			if err != nil {
-				uis.LoggedError(w, r, http.StatusInternalServerError, err)
+				grip.Error(message.Fields{
+					"message": "fetching execution task",
```

Comment:
`fetching test results for execution task`

Reply:
fixed